NUModules/NUModule.j

```diff
@@ -1140,7 +1142,13 @@ NUModuleTabViewModeIcon                = 2;
     if (!_isProcessingPush)
         [self showLoading];
 
-    if ([[self class] automaticContextManagement])
+    if (_usesPagination && [_categories count])
```

Comment:
If no context is defined in the category everything should work like before, that means no pagination. In this way we can merge without breaking all the NUModules that use categories.